code/clue.js

```diff
@@ -1,59 +1,319 @@
   lastName: 'Green',
-  color: 'green',
+  color: '#16a83d',
   description: 'He has a lot of connections',
   age: 45,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
+  occupation: 'Entrepreneur',
+  favoriteWeapon: 'knife'
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const prPlum = {
+  firstName: 'Peter',
+  lastName: 'Plum',
+  color: '#ff40ef',
+  description: 'He is highly intelligent',
+  age: 36,
+  image: 'assets/plum.png',
+  occupation: 'Professor',
+  favoriteWeapon: 'bat'
+}
+
+const msScarlet = {
+  firstName: 'Josephine',
+  lastName: 'Scarlet',
+  color: '#ff3030',
+  description: 'She is always wearing red',
+  age: 25,
+  image: 'assets/scarlet.png',
+  occupation: 'Actress',
+  favoriteWeapon: 'poison'
+}
+
+const mrsPeacock = {
+  firstName: 'Elizabeth',
+  lastName: 'Peacock',
+  color: '#4050ff',
+  description: 'She enjoys ballroom dancing',
+  age: 32,
+  image: 'assets/peacock.png',
+  occupation: 'Socialite',
+  favoriteWeapon: 'candlestick'
+}
+
+const colMustard = {
+  firstName: 'Michael',
+  lastName: 'Mustard',
+  color: '#f0fa32',
+  description: 'He has a stiff right leg and walks with a limp',
+  age: 60,
+  image: 'assets/mustard.png',
+  occupation: 'Colonel',
+  favoriteWeapon: 'trophy'
+}
+
+const mrsWhite = {
+  firstName: 'Blanche',
+  lastName: 'White',
+  color: 'white',
+  description: 'She likes her surroundings spotless',
+  age: 70,
+  image: 'assets/white.png',
+  occupation: 'Housekeeper',
+  favoriteWeapon: 'pistol'
+}
+
+// WEAPON OBJECTS
 
 const rope = {
   name: 'rope',
-  weight: 10
+  weight: 0.3,
+  id: 'rope'
+}
+
+const knife = {
+  name: 'knife',
+  weight: 1,
+  id: 'knife'
+}
+
+const candlestick = {
+  name: 'candlestick',
+  weight: 2,
+  id: 'candlestick'
+}
+
+const dumbbell = {
+  name: 'dumbbell',
+  weight: 16,
+  id: 'dumbbell'
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const poison = {
+  name: 'poison',
+  weight: 0.2,
+  id: 'poison'
+}
+
+const axe = {
+  name: 'axe',
+  weight: 7,
+  id: 'axe'
+}
+
+const bat = {
+  name: 'bat',
+  weight: 3,
+  id: 'bat'
+}
+
+const trophy = {
+  name: 'trophy',
+  weight: 15,
+  id: 'trophy'
+}
+
+const pistol = {
+  name: 'pistol',
+  weight: 2,
+  id: 'pistol'
+}
+
+// ROOM VARIABLES
+
+const roomDining = 'Dining Room';
+
+const roomConservatory = 'Conservatory';
+
+const roomKitchen = 'Kitchen';
+
+const roomStudy = 'Study';
+
+const roomLibrary = 'Library';
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const roomBilliard = 'Billiard Room';
+
+const roomLounge = 'Lounge';
+
+const roomBallroom = 'Ballroom';
+
+const roomHall = 'Hall';
+
+const roomSpa = 'Spa';
+
+const roomLiving = 'Living Room';
+
+const roomObservatory = 'Observatory';
+
+const roomTheather = 'Theater';
+
+const roomGuest = 'Guest House';
+
+const roomPatio = 'Patio';
+
+// OBJECT THAT KEEPS THE MYSTERY.
+
+let mystery = {
+  killer: '',
+  weapon: '',
```

Comment:
It's even more semantic approach to initialise mystery object with `null` values like this :

```
const mystery = {
  killer: null,
  room: null,
  weapon: null
}
```

Moreover, you can use `const` keyword - we will be changing values of properties, that's true, but JavaScript will not consider this as overwriting. What counts for JavaScript is that object will stay object